src/components/editor-page/renderer-pane/hooks/use-on-iframe-load.ts

```diff
@@ -6,6 +6,9 @@
 
 import { RefObject, useCallback, useRef } from 'react'
 import { EditorToRendererCommunicator } from '../../../render-page/window-post-message-communicator/editor-to-renderer-communicator'
+import { Logger } from '../../../../utils/logger'
+
+const log = new Logger('IFrameLoader')
```

Comment:
```suggestion
const log = new Logger('IframeLoader')
```

See line 13 (old 10) for reference.